app/templates/events/view/videoroom/list.hbs

```diff
@@ -26,12 +26,8 @@
   <div class="ui grid space-between items-center mt-4 mb-4">
       <div class="d-flex flex-row">
         <h2 class="header">{{t 'Microlocation Video Rooms'}}</h2>
-        <UiPopup
-          @content={{t 'Microlocation video rooms are suitable for events with different tracks, rooms and sessions that are listed in a schedule. All microlocations are listed on the event schedule.'}}
-          class="m-2" >
-          <i class="info circle icon"></i>
-        </UiPopup>
       </div>
+            <p>{{t 'Microlocation video rooms are suitable for events with different tracks, rooms and sessions that are listed in a schedule. All microlocations are listed on the event schedule. Note: Microlocation rooms require a session in order to be visible to attendees. They are only listed on the video channel list on side panels of online events, if session have been added into the location in the schedule.'}}</p>
```

Comment:
@iamdevvalecha This is probably the third time I am reviewing one of your PRs just to tell you to fix the indentation. You need to follow up and understand what the review says so that we can avoid those errors in the future PRs. I wonder how you can miss the indentation when the PR contains only a one line change. If you have a code formatter working automatically on save, kindly disable it for this project.

Reply:
sorry, i think i need to switch to vs code back.